packages/disploy/src/structs/User.ts

```diff
@@ -28,9 +28,12 @@ export class User extends Base {
 	 * @example '0005'
 	 */
 	public discriminator!: string;
-
+	/**
+	 * The raw data of the user.
+	 */
+	declare public raw: APIUser
 	public constructor(app: App, raw: APIUser) {
-		super(app);
+		super(app, raw);
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.